documentation/text-utilities.html

```diff
@@ -0,0 +1,183 @@
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Exo:wght@300;500;600;700&display=swap" />
+    <link rel="stylesheet" href="../gentle-imports.css" />
+    <link rel="stylesheet" href="documentation.css" />
+    <link rel="icon" href="https://img.icons8.com/stickers/100/000000/g.png " />
+    <title>Text Utilities | Gentle UI</title>
+</head>
+
+<body>
+    <div class="documentation-layout">
+        <div class="navbar p-4">
+            <div class="nav-left p-4">
+                <img src="https://img.icons8.com/stickers/60/000000/g.png" />
+                <h5><a href="../index.html" class="link-none">Gentle UI</a></h5>
+            </div>
+
+            <ul class="nav-right p-4">
+                <li>
+                    <a href="../index.html" class="link-none">HOME</a>
+                </li>
+                <li>
+                    <a href="https://github.com/dev-enforced/Gentle_UI" class="link-none">
+                        <i class="fab fa-github fa-2x"></i>
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <div class="sidebar p-6">
+            <h4 class="text-bold">COMPONENTS</h4>
+            <ul class="sidebar-components">
+                <li class="p-2 my-2">
+                    <h6><a class="link-none" href="alerts.html">ALERT</a></h6>
+                </li>
+                <li class="p-2 my-2">
+                    <h6><a class="link-none" href="avatars.html">AVATARS</a></h6>
+                </li>
+                <li class="p-2 my-2">
+                    <h6><a class="link-none" href="badges.html">BADGES</a></h6>
+                </li>
+                <li class="p-2 my-2">
+                    <h6><a class="link-none" href="buttons.html">BUTTONS</a></h6>
+                </li>
+                <li class="p-2 my-2">
+                    <h6><a class="link-none" href="cards.html">CARDS</a></h6>
+                </li>
+                <li class="p-2 my-2">
+                    <h6><a class="link-none" href="grids.html">GRIDS</a></h6>
+                </li>
+                <li class="p-2 my-2">
+                    <h6><a class="link-none" href="images.html">IMAGES</a></h6>
+                </li>
+                <li class="p-2 my-2">
+                    <h6><a class="link-none" href="inputs.html">INPUT</a></h6>
+                </li>
+                <li class="p-2 my-2">
+                    <h6><a class="link-none" href="lists.html">LISTS</a></h6>
+                </li>
+                <li class="p-2 my-2">
+                    <h6><a class="link-none" href="navigation.html">NAVIGATION</a></h6>
+                </li>
+                <li class="p-2 my-2">
+                    <h6>
+                        <a class="link-none" href="text-utilities.html">TEXT UTILITIES</a>
+                    </h6>
+                </li>
+            </ul>
+        </div>
+        <div class="main-content p-5">
```

Comment:
You can use a `main` tag here

Reply:
Yes will use keep in mind while making the upcoming pages of the project